src/apps/project-collaboration/InviteUser/InviteUser.tsx

```diff
@@ -36,162 +38,167 @@ export const InviteUser = (props: InviteUserProps) => {
 
   const [error, setError] = useState<string | undefined>(undefined);
 
-  const invitedBy = me.nickname ? 
-    me.nickname : (me.first_name || me.last_name) ?
-    [me.first_name, me.last_name].join(' ') :
-    undefined;
+  const invitedBy = me.nickname
+    ? me.nickname
+    : me.first_name || me.last_name
+    ? [me.first_name, me.last_name].join(' ')
+    : undefined;
 
   const setOpen = (open: boolean) => {
     formik.resetForm();
     setError(undefined);
     _setOpen(open);
-  }
+  };
 
   const sendInvitation = (email: string, group: string) => {
     setBusy(true);
 
     // Waits until the invite was processed in the backend
-    const a = new Promise(resolve => {
-      inviteUserToProject(supabase, email, project, group, invitedBy)
-        .then(({ error, data }) => {
-          if (error) {
-            props.onInvitiationError(error);
+    const a = new Promise((resolve) => {
+      inviteUserToProject(supabase, email, project, group, invitedBy).then(
```

Comment:
I saw that `inviteUserToProject` uses a `fetch` to call an API method. Is there a reason the email isn't sent from the `inviteUserToProject` method directly, without going through a server-side API call?

Reply:
It uses some secrets to send the email.